### PR TITLE
Replace custom adapters with public adapters library

### PR DESCRIPTION
**Pull request by pshevche**
## Summary

See https://github.com/gradle/develocity-agent-adapters

### CODE REVIEW

**Review by jean-andre-gauthier — APPROVED** [review]

**Review by jprinet — APPROVED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -64,10 +45,13 @@ java {
     }
 }
 
-jar {
-    from sourceSets.compatibilityApi.output
-    from sourceSets.develocityCompatibility.output
-    from sourceSets.enterpriseCompatibility.output
+shadowJar {
+    // required by the plugin-publish-plugin
+    archiveClassifier = ''
```
Comment (runningcode) [inline build.gradle:50]: do the regular `jar` task and the `shadow` jar task now produce output to the same location? @pshevche

Comment (pshevche) [reply build.gradle:50]: Yes, they do

Comment (runningcode) [reply build.gradle:50]: Can we fix that? Overlapping outputs are a problem for correctness/reproducability. We will either have a `regular jar` or a `shadow jar` depending on the ordering of the tasks in the build.

Comment (pshevche) [reply build.gradle:50]: Do you know how to fix this? I can look into this with some pointers.

Comment (runningcode) [reply build.gradle:50]: I see two possibilities, 1. give it a different classifier, 2. disable the jar task. Maybe the BT team has more ideas though.

**Pull request merged**